set_env.sh

```diff
@@ -36,7 +36,7 @@ case `uname` in
             if [ -z "$CPP_CROSS_COMPILE" ] ; then
                 BUNDLE_PLATFORM=linux-x86_64
             else
-                if [ "$CPP_CROSS_COMPILE" = macosx ] ; then
+                if [ "$CPP_CROSS_COMPILE" = macos ] ; then
```

Comment:
This will also require a change to `CPP_CROSS_COMPILE` in `dev-tools//docker/macosx_builder/Dockerfile`

Reply:
There are actually quite a few places I'll have to change to switch `CPP_CROSS_COMPILE` from using `macosx` to `macos`, and it will introduce an annoying discrepancy between branches in the Jenkins infrastructure.  So I think I'll revert this one place in this PR.  Maybe a future PR can do the rename fully.